app/models/appointment.rb

```diff
@@ -1,7 +1,10 @@
 class Appointment < ApplicationRecord
   include PgSearch
 
-  pg_search_scope :search, against: %i(id first_name last_name)
+  pg_search_scope :search, against: %i(id first_name last_name), associated_against: {
+    guider: [:name]
```

Comment:
@benlovell it's not case sensitive, and afaik does porter stemming. Some more specs might be warranted.

It does *not* do partial matches, and actually after some testing I haven't seen it do porter stemming either.